requirements/anomaly.txt

```diff
@@ -1,4 +1,6 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # Anomaly Requirements.
+torchvision==0.14.1
```

Comment:
Do we need this, too?

Reply:
Yes, without this, torchvision==0.15.1 and torch==2.0.0 are installed.

Reply:
What if we fix the torch version instead and let the installer sort out the torchvision and torchtext versions?

Reply:
> What if we fix the torch version instead and let the installer sort out the torchvision and torchtext versions?

Currently, we regard the torch as pre-requisite. Users might want to use torch1.12.1. I would prefer @ashwinvaidya17's approach.

Reply:
umm.. I'm also not exactly sure this requirement changes. This PR is hotfix.
If we can control the acceptable range of version in anomaly side, it looks best to me. https://github.com/openvinotoolkit/anomalib/blob/v0.4.0rc2/requirements/base.txt#L15-L16

Reply:
![image](https://user-images.githubusercontent.com/4629606/225674972-267262a8-74a5-425e-a661-2dec55e807d8.png)